dnf-docker-test/features/deplist-1.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Deplist as commmand and option
+
+  Scenario: Feature Setup
+      Given I use the repository "upgrade_1"
```

Comment:
Less work and now idea how to setup more complicate repo, with versions, and requires. Pity that you pushed the changes, because they are not mature.